documentation/asciidoc/computers/camera/external_trigger.adoc

```diff
@@ -0,0 +1,53 @@
+== External Trigger
+=== IMX296
```

Comment:
Perhaps this should be titled "External Trigger on the Raspberry Pi Global Shutter Camera" and remove the IMX296 subsection?